test/evergreen.yml

```diff
@@ -2490,6 +2496,20 @@ tasks:
             export "PATH=/opt/mongodbtoolchain/v3/bin:$PATH"
             ./workgen_perf_check.sh many-dhandle-stress.py output.log 50 500 500
 
+  - name: many-collection-test
+    commands:
+      - func: "fetch mongodb bin"
+      - func: "fetch mongo-tests repo"
+      - command: shell.exec
+        params:
+          working_dir: mongo-tests/largescale
+          script: |
+            set -o errexit
+            set -o verbose
+            export "PATH=/opt/mongodbtoolchain/v3/bin:$PATH"
```

Comment:
The PATH setting is usually covered at the build variant level.

Reply:
I remember we had a chat about it when we implemented the task `many-dhandle-stress-test` and had to set the PATH before executing the script since `python` is executed within the script itself.

Reply:
I see, thanks for the explanation.